packages/adena-extension/src/common/validation/validation-wallet.ts

```diff
@@ -0,0 +1,8 @@
+export function isSeedPhraseString(seedPhrase: string, length: number): boolean {
```

Comment:
Just curious, do we ever check for the validity of the mnemonic?
Obviously not here, but in general

Reply:
Normally, when creating a mnemonic, it would be checked if it is a valid mnemonic. 
- Whether it is included in a BIP39 compliant managed wordlist.
- Number of words, EnglishMnemonic in cosmjs supports 12, 15, 18, 21, 24.

It will be validated when creating the wallet for the tm2-js-client.
https://github.com/gnolang/tm2-js-client/blob/main/src/wallet/utility/utility.ts#L56

The function on this line is a function that checks the number of words the user has entered,
Now that I think about it, I should change the function name to indicate that it checks for word length.